hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestDeleteBlocksCommandHandler.java

```diff
@@ -262,6 +263,23 @@ public List<Future<DeleteBlockTransactionExecutionResult>> answer(
         blockDeleteMetrics.getTotalLockTimeoutTransactionCount());
   }
 
+  @Test
+  public void testDeleteCmdWorkerInterval() {
+    OzoneConfiguration tmpConf = new OzoneConfiguration();
+    tmpConf.setLong(BLOCK_DELETE_COMMAND_WORKER_INTERVAL, 3000);
+    OzoneContainer container = Mockito.mock(OzoneContainer.class);
+    DatanodeConfiguration dnConf =
+        tmpConf.getObject(DatanodeConfiguration.class);
+    DeleteBlocksCommandHandler commandHandler =
+        spy(new DeleteBlocksCommandHandler(
+        container, tmpConf, dnConf, "test"));
+
+    Assert.assertEquals(Long.parseLong(tmpConf.get(
+        BLOCK_DELETE_COMMAND_WORKER_INTERVAL)), 3000);
```

Comment:
Since the test has set this value earlier, this assertion does not exercise the `DeleteCmdWorker` or any other production code.

```suggestion
```